src/seasideimport.h

```diff
@@ -46,7 +47,7 @@ class CONTACTCACHE_EXPORT SeasideImport
     ~SeasideImport();
 
 public:
-    static QList<QContact> buildImportContacts(const QList<QVersitDocument> &details, int *newCounti = 0, int *updatedCount = 0);
+    static QList<QContact> buildImportContacts(const QList<QVersitDocument> &details, int *newCounti = 0, int *updatedCount = 0, int *ignoredCount = 0, SeasideContactBuilder *builder = 0);
```

Comment:
Might as well fix 'newCounti' while you're here.

Also, may be better to preserve BC by adding an overload that has only builder defaulted, and having the existing function call the new one with defaults.